patternfly/patternfly-elements
# chore: Update generator with knobs import; add events to templates

## Infrastructure updates

Updating the generator to support events, readme injection, and fixing the reference to the add-on knobs import which no longer requires the `polymer` classification.

### Testing instructions

- [x] `npm run new`
  - Be sure to add a few events
  - Check the compiled storybook result renders with no errors


### Ready-for-merge Checklist




### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
